server/api/contest_handler.py

```diff
@@ -1,7 +1,9 @@
 import json
 from flask import jsonify, Blueprint, request
 from api import db
-from api.models import Contest, Submission, User
+from models.contest_model import Contest

```

Comment:
In your models folder, if you add in a `__init__.py` file, you would be able to import anything inside this folder in a one liner such as 

`from models import Contest, Submissions, User`